jest/jest.d.ts

```diff
@@ -1,126 +1,88 @@
-// Type definitions for Jest 0.9.0
+// Type definitions for Jest 15.1.1
 // Project: http://facebook.github.io/jest/
-// Definitions by: Asana <https://asana.com>
+// Definitions by: Asana <https://asana.com>, Ivo Stratev <https://github.com/NoHomey>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference path="../node/node.d.ts" />
 
-declare function afterEach(fn: jest.EmptyFunction): void;
-declare function beforeEach(fn: jest.EmptyFunction): void;
-declare function describe(name: string, fn: jest.EmptyFunction): void;
-declare var it: jest.It;
-declare function pit(name: string, fn: jest.EmptyFunction): void;
-
-declare function xdescribe(name: string, fn: jest.EmptyFunction): void;
-declare function xit(name: string, fn: jest.EmptyFunction): void;
-
+declare function afterEach(fn: () => any): void;
+declare function beforeEach(fn: () => any): void;
```

Comment:
They are not documented, see https://facebook.github.io/jest/docs/api.html#content. Jest are slowly replacing Jasmine with their own implementation, see https://facebook.github.io/jest/blog/#new-cli-error-messages-and-summaries. So I guest they are better off without definitions.